### PR TITLE
getting overzealous with the deprecations

### PR DESCRIPTION
**Pull request by makosblade**

### CODE REVIEW

**Comment by makosblade** [comment]
@UnsungHero97 you were right! I derped

File: src/main/java/com/kryptnostic/storage/v1/models/ObjectMetadata.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
  *
  * @author sinaiman, rbuckheit
  */
+@Deprecated
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/storage/v1/models/ObjectMetadata.java:24]: just confirming... this is being deprecated in favor of `com.kryptnostic.v2.storage.models.ObjectMetadata`?

**Comment by UnsungHero97** [comment]
ship it!

File: src/main/java/com/kryptnostic/storage/v1/models/ObjectMetadata.java (hunk 1 above)
Comment (makosblade) [inline src/main/java/com/kryptnostic/storage/v1/models/ObjectMetadata.java:24]: Yeah. We actually don't use it anywhere but I didn't want to pull it all out right now

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 2]
```diff
@@ -34,16 +34,16 @@
     String VERSION                    = "version";
     String BLOCK                      = "block";
     String TYPE_NAME_FIELD            = "type";
-    String ACL_ID_VAR                    = "aclid";
-    String PAGE_SIZE_VAR                 = "pageSize";
-    String LATEST_CLOCK_VAR              = "latestClock";
+    String ROOT_OBJECT_ID_VAR                    = "aclid";
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java]: spacing

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 3]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
...
             ObjectTreeLoadRequest request );
 
     @POST( CONTROLLER + FULL_LEVELS_NEXT_PAGE_PATH )
     ObjectTreeLoadResponse continueObjectsByTypeAndLoadLevelPaged(
-            @Path( ACL_ID_VAR ) UUID rootAclId,
-            @Path( PAGE_SIZE_VAR ) int pageSize,
-            @Path( LATEST_CLOCK_VAR ) UUID latestClock,
+            @Path( ROOT_OBJECT_ID_VAR ) UUID rootAclId,
+            @Path( SCROLL_SIZE_VAR ) int pageSize,
+            @Path( LATEST_OBJECT_ID_VAR ) UUID latestClock,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java]: `latestClock` -> `latestObjectId`

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 4]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
...
             @Path( VERSION ) long latestClockVersion,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java]: `latestClockVersion` -> `latestObjectVersion`

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 5]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
 
     @POST( CONTROLLER + FULL_LEVELS_INITIAL_PAGE_PATH )
     ObjectTreeLoadResponse getObjectsByTypeAndLoadLevelPaged(
-            @Path( ACL_ID_VAR ) UUID rootAclId,
-            @Path( PAGE_SIZE_VAR ) int pageSize,
+            @Path( ROOT_OBJECT_ID_VAR ) UUID rootAclId,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java:189]: `rootAclId` -> `rootObjectId` or `rootId`

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 6]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
...
+            @Path( SCROLL_SIZE_VAR ) int pageSize,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java]: `pageSize` -> `scrollSize`

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 7]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
...
             ObjectTreeLoadRequest request );
 
     @POST( CONTROLLER + FULL_LEVELS_NEXT_PAGE_PATH )
     ObjectTreeLoadResponse continueObjectsByTypeAndLoadLevelPaged(
-            @Path( ACL_ID_VAR ) UUID rootAclId,
-            @Path( PAGE_SIZE_VAR ) int pageSize,
-            @Path( LATEST_CLOCK_VAR ) UUID latestClock,
+            @Path( ROOT_OBJECT_ID_VAR ) UUID rootAclId,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java:195]: `rootAclId` -> `rootObjectId` or `rootId`

File: src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java [hunk 8]
```diff
@@ -186,15 +186,15 @@ Response setObjectFromBlockCiphertext(
...
+            @Path( SCROLL_SIZE_VAR ) int pageSize,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/ObjectStorageApi.java]: `pageSize` -> `scrollSize`

**Pull request merged**